### DISCUSSION TITLE
咨询一下每半小时就会自动重新登录正常吗

### DISCUSSION THREAD

**Discussion by zmdyy0318** [mamoe/mirai#971] [General]
怀疑是这个导致的风控
阿里云杭州，学生机ecs
![IMG_7055](https://user-images.githubusercontent.com/16699676/106998028-d6cae380-67be-11eb-8e7f-ab8647f10e4c.PNG)

**Comment by ryoii** [comment]
正常，你应该是没有使用固定的 deviceInfo

**Reply by zmdyy0318** [reply]
我查看了目录下device.json是正常的，我的交互方式是cpp→mirai-http。大佬可以说下如何使用固定deviceinfo嘛？

**Reply by ryoii** [reply]
使用console应该是固定的device。我没有找到这个页面在什么地方，就目前使用反馈来说，30分钟进行一次重新登录并不会造成风控

**Reply by zmdyy0318** [reply]
好的谢谢
